voice-adapter.js

```diff
@@ -328,17 +327,16 @@ function loadVoiceAdapter(addonManager, manifest, _errorCallback) {
 }
 
 function checkInstallation() {
-  const installation_checker_process = spawn('dpkg-query', ['-s', 'snips-platform-voice']);
```

Comment:
So the the reasoning for removing the check for the snips installation here is that we should always try to install the whole package whenever the user enables the addon? Actually I like this way better.

Reply:
I moved the check into the script itself, that way we can more easily support other distributions.